easybuild/easyconfigs/a/ABINIT/ABINIT-7.6.2-foss-2015a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = "ConfigureMake"
+
+name = 'ABINIT'
+version = '7.6.2'
+
+homepage = 'http://www.abinit.org/'
+description = """Abinit is a plane wave pseudopotential code for doing
+ condensed phase electronic structure calculations using DFT."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
```

Comment:
@AFCJamie same as in #3449, lots of lines with trailing whitespace, please clean that up?